### ISSUE TITLE
Get rid of references to RAN in corroborate_monument_data.py

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rowiki/wikiro#19]
```
We should create a generic way of interpreting known formats (i.e. field 
conversion or similar).

This is low priority since we have a command line option that prevents RAN 
database processing.
```

Original issue reported on code.google.com by `strain...@gmail.com` on 10 Feb 2015 at 11:35
- Blocking: #17
- Blocked on: #29
